pkg/services/datasources/cache.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 type CacheService interface {
-	GetDatasource(datasourceID int64, user *models.SignedInUser, skipCache bool) (*models.DataSource, error)
+	GetDatasource(datasourceID int64, orgId int64, skipCache bool) (*models.DataSource, error)
```

Comment:
This change will break Enterprise

Reply:
I think I can revert it, thanks for noticing!